buildspec.yml

```diff
@@ -1,26 +1,115 @@
 version: 0.2
 
-# Running as the root user as the default.
-#run-as: Linux-user-name
-
 env:
-  # use bash as the default shell
   shell: bash
 
 phases:
   pre_build:
     commands:
       # Set up the environment for trustzone backend
       - pushd /work/rust-optee-trustzone-sdk && source environment && popd
-      # clean the repostory, which automatically downloads the rust toolchain specified in the repostory
+      # Clean the repostory, which automatically downloads the rust toolchain specified in the repostory
       - make clean
   build:
     commands:
-      # compiling and running tests on trustzone
+      # Compile and run tests on trustzone
       - make trustzone-durango-test
       - make trustzone-sinaloa-test
       - make trustzone-veracruz-test
-      # compiling and running some tests on sgx; other tests requires SGX enable machine.
+      # Compiling and run durango tests on sgx; other tests require SGX enable machine.
       - make sgx-durango-test
       - make sgx-sinaloa-test-dry-run
       - make sgx-veracruz-test-dry-run
+
+
+
+#---------------------------------------
```

Comment:
Should we document this in here?  This seems like something that's better suited for the project wiki?

Reply:
I actually don't know. I don't think it makes sense to document in project wiki, because it is more like a development log for any maintainers. It is also AWS specific. That is why I documented in buildspec.yml which is only for AWS.

Reply:
OK, that sounds reasonable.